packages/dcos-integration-test/extra/resiliency/test_aws_cf.py

```diff
@@ -0,0 +1,75 @@
+import logging
+import os
+
+import pytest
+import requests
+import retrying
+import test_util.aws
+
+
+@pytest.fixture(scope='session')
+def provider(cluster):
+    """Interface for direct interation to provider hardware
+    Currently only supports AWS CF with AWS VPC coming soon
+    """
+    if cluster.provider != 'aws':
+        pytest.skip('Must be AWS CF to run test')
+    aws_access_key_id = os.environ['AWS_ACCESS_KEY_ID']
+    aws_secret_access_key = os.environ['AWS_SECRET_ACCESS_KEY']
+    aws_region = os.environ['AWS_REGION']
+    stack_name = os.environ['AWS_STACK_NAME']
+    bw = test_util.aws.BotoWrapper(aws_region, aws_access_key_id, aws_secret_access_key)
+    return test_util.aws.DcosCfSimple(stack_name, bw)
+
+
+@retrying.retry(wait_fixed=3000, stop_max_delay=300 * 1000)
+def wait_for_persistent_app(url):
```

Comment:
Actually, there is no cluster specific code in here and its not a fixture so it doesnt belong in conftest, but what I could do is make it a function returned from the fixture which has the sole purpose of checking connectivity to these two apps, but that is somewhat convoluted when we could have a simple fixture and a simple function instead, right?